.github/workflows/release.yml

```diff
@@ -34,25 +34,11 @@ jobs:
             .cargo/git/db/
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-      - name: Use Nix
-        uses: cachix/install-nix-action@v20
-        with:
-          nix_path: nixpkgs=channel:nixos-21.05
-      - name: Configure Nix substituters
-        run: |
-          set -xe
-          mkdir -p ~/.config/nix/
-          cp ./.github/nix.conf ~/.config/nix/
-      - name: Use cachix
-        uses: cachix/cachix-action@v12
-        with:
-          name: holochain-ci
-      - name: Prepare Nix environment
-        run: nix-shell --command "echo Completed"
```

Comment:
these changes are good, but they also need to remove lines 24-36. 

Also, you need to use the rust toolchain for this job instead of the create-release one. 
Make sure to also bring `targets`